Tests/QuantConnect.Tests.csproj

```diff
@@ -304,6 +304,12 @@
     <None Update="TestData\spy_arima.csv">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="TestData\spy_si.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="TestData\spy_asi.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
> Test data was sourced from the "spy_ad.txt" file included in the TestData folder.

Where was the original source of the data ( i.e. a third party source or self-generated? )  We need all indicators to be vetted against a third-party source as the community routinely sanity checks the indicator results, its always good to know in advance its been compared and passed.

Reply:
Fixed in fe76c48 and f016058. After a brief look around, I couldn't find any info on the original source of "spy_ad.txt", so I simply replaced the test data sets for SI and ASI with data sourced from TradingView.

In the process, I also discovered that the R value for SI was being calculated incorrectly. This issue was fixed in 9b41f50.